projects/github-actions/repo-gardening/README.md

```diff
@@ -1,17 +1,93 @@
-# GitHub repository gardening
+# GitHub Repository Gardening Action
 
-This action includes multiple different tasks to automate some of the things we must do to each issue and PR that is handled in the repo.
+This GitHub Action includes multiple tasks, to automate some of the things we must do to each issue and PR that is handled in an Automattic repository.
+
+**Note**: this action addresses needs and uses standards that are specific to Automattic. They may not be useful to you.
 
 Here is the current list of tasks handled by this action:
 
-- Assign Issues: Adds assignee for issues which are being worked on, and adds the "In Progress" label.
-- Add Milestone: Adds a valid milestone to all PRs that get merged and don't already include a milestone.
-- Check Description: Checks the contents of a PR description, and ensure it matches our recommendations.
-- Add Labels: Adds labels to PRs that touch specific features.
-- WordPress.com Commit Reminder: Posts a comment on merged PRs to remind Automatticians to commit the matching WordPress.com change.
+- Assign Issues (`assignIssues`): Adds assignee for issues which are being worked on, and adds the "In Progress" label.
+- Add Milestone (`addMilestone`): Adds a valid milestone to all PRs that get merged and don't already include a milestone.
+- Check Description (`checkDescription`): Checks the contents of a PR description, and ensure it matches our recommendations.
+- Add Labels (`addLabels`): Adds labels to PRs that touch specific features.
+- Clean Labels (`cleanLabels`): Removes Status labels once a PR has been merged.
+- WordPress.com Commit Reminder (`wpcomCommitReminder`): Posts a comment on merged PRs to remind Automatticians to commit the matching WordPress.com change.
+- Notify Design (`notifyDesign`): Sends a Slack Notification to the Design team to request feedback, based on labels applied to a PR.
+
+Some of the tasks are may not satisfy your needs. If that's the case, you can use the `tasks` option to limit the action to the list of tasks you need in your repo. See the example below to find out more.
+
+## Usage
+
+### Example
+
+In the example below, we'll set up the action to run 2 tasks (`cleanLabels` and `notifyDesign`):
+
+```yml
+name: Clean Labels and notify design
```

Comment:
If someone actually uses this, the resulting name in the GH UI will be "Clean Labels and notify design / Clean up labels, and notify Design when necessary". I wonder if we should change the workflow name to just "Repo Gardening" or something like that that people could more sensibly add things to.
```suggestion
name: Repo Gardening
```